Task1.js

```diff
@@ -0,0 +1,27 @@
+function GroupByKey(array,key)
+{
+  var tempData = {};
+
+  for ( var index=0; index<array.length; index++ )
```

Comment:
The logic works fine and is good.
But I want you to write another function that uses array.reduce method to do the same thing.

Reply:
Sure sir